insights/core/plugins.py

```diff
@@ -276,6 +276,8 @@ def _visit_parsers(parent, visitor):
 
 def get_dependency_graph(component):
     graph = defaultdict(set)
+    if not COMPONENT_DEPENDENCIES[component]:
```

Comment:
Is this code that I had somehow removed?  It's not apparent why this was added in this PR.